lib/binary_to_decimal.rb

```diff
@@ -5,5 +5,25 @@
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
+  array = binary_array
+  power = 7
```

Comment:
Minor:  I suggest making 7 a constant or using `binary_array.length -1` just to make things more flexible and avoid hardcoding randomish numbers.